test/solc/CommandLineParser.cpp

```diff
@@ -175,7 +175,7 @@ BOOST_AUTO_TEST_CASE(cli_mode_options)
 		expectedOptions.output.dir = "/tmp/out";
 		expectedOptions.output.overwriteFiles = true;
 		expectedOptions.output.evmVersion = EVMVersion::spuriousDragon();
-		expectedOptions.output.experimentalViaIR = true;
+		expectedOptions.output.viaIR = true;
```

Comment:
There's `--experimental-via-ir` higher up, in the `vector<string> commandLine` initializer. I'd actually add `--via-ir` next to it, instead of replacing it there - both flags are still supported after all and this test is supposed to use all possible flags.